app/controllers/Application.java

```diff
@@ -682,6 +687,13 @@ private static Result getJobHistory(Version version) {
     boolean hasSparkJob = false;
     // get the graph type
     String graphType = form.get("select-graph-type");
+    
+    SqlQuery q = Ebean.createSqlQuery("select distinct job_def_id from yarn_app_result order by job_def_id;");
```

Comment:
I am not in favor of merging this. In prod environments, there will be millions of entries and this can become an overkill.